rotkehlchen/chain/ethereum/modules/constants.py

```diff
@@ -1,3 +1,4 @@
+from rotkehlchen.chain.ethereum.modules.sushiswap.constants import CPT_SUSHISWAP_V2
```

Comment:
I think `CPT_SUSHISWAP_V2` should be added to `AMM_POSSIBLE_COUNTERPARTIES` since 1inch also supports Sushiswap

Reply:
indeed, thanks